paper/main.tex

```diff
@@ -1487,7 +1475,10 @@ \section*{Author contributions}
 Concept: J.R.M.  Implementation: T.H.C., L.L.W.O., and J.R.M.
 Analyses: L.L.W.O. and J.R.M.  Writing: L.L.W.O. and J.R.M.
 
-\bibliographystyle{apacite}
+\section*{Competing interests}
+The authors declare no competing financial interests.
+
+\bibliographystyle{naturemag}
 \bibliography{CDL-bibliography/cdl}
```

Comment:
Once we have everything else together for the paper, you can paste in contents of main.bib directly (replace lines 1481 and 1482 with the contents of main.bib, once the formatting looks good)

Reply:
okey doke